examples/combobox/js/combobox-autocomplete.js

```diff
@@ -57,7 +57,12 @@ class ComboboxAutocomplete {
       'focus',
       this.onComboboxFocus.bind(this)
     );
-    this.comboboxNode.addEventListener('blur', this.onComboboxBlur.bind(this));
+
+    document.body.addEventListener(
+      'mouseup',
```

Comment:
One note, mouseup fire when dragging the scrollbar, blur doesn't.

Reply:
Is there another issue to resolve here? Does that mean that scrolling the page unexpectedly closes the dropdown? As a screen reader user, I can't test this, so I'm not quite clear on what the problem is.

Reply:
I was raising a regression (not sure about the importance of it) on the UX for mouse users. I'm not aware of any impact on screen readers.